google-cloud-bigquery/lib/google/cloud/bigquery/dataset/access.rb

```diff
@@ -277,7 +279,7 @@ def add_writer_domain domain
           # Add writer access to a special group.
           #
           # @param [String] group Accepted values are `owners`, `writers`,
-          #   `readers`, and `all`.
+          #   `readers`, `all_authenticated_users`, and `all_users`.
```

Comment:
Could you also update the code examples used in these methods to use the `:all_users` value instead of `:all`?

```ruby
          #   dataset.access do |access|
          #     access.add_writer_special :all_users
          #   end
```

Reply:
updated code examples